app/models/host/discovered.rb

```diff
@@ -178,4 +182,10 @@ def compute_resource
     false
   end
 
+  def lookup_value_match
```

Comment:
@domcleal i can see value in a baseline method for the STI classes to override, yeah. I think it's harmless to leave this here for now, as we genuinely don't want to do matches on Discovered Hosts (for now anyway). I'll try and send something to core next week.